examples/panda-css/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "panda-css",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "prepare": "npm run codegen",
+    "codegen": "panda codegen",
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@pandacss/dev": "^0.13.1",
```

Comment:
Could we update all of these to their latest versions?

Reply:
all dependencies updated